src/components/Lane.js

```diff
@@ -204,7 +205,7 @@ class Lane extends Component {
           getChildPayload={index => this.props.getCardDetails(id, index)}>
           {cardList}
         </Container>
-        {editable && !addCardMode && this.renderAddCardLink()}
+        {editable && selfEditable && !addCardMode && this.renderAddCardLink()}
```

Comment:
There's no need for this to be `selfEditable` - instead, just call it `editable`, and refactor the current `editable` *variable* to be `boardEditable` (the prop can stay the same - it's just the local variable name that needs to change).

Reply:
done